src/Avalonia.Controls/TopLevel.cs

```diff
@@ -285,6 +285,9 @@ IInputElement IInputRoot.PointerOverElement
         /// <inheritdoc/>
         IMouseDevice IInputRoot.MouseDevice => PlatformImpl?.MouseDevice;
 
+        /// <inheritdoc/>
+        bool? IInputRoot.IsPlatformEnabled => PlatformImpl is IWindowImpl windowImpl? windowImpl.IsEnabled : (bool?)null;
```

Comment:
Could be shortened to `(PlatformImpl as IWindowImpl)?.IsEnabled` I think?

Reply:
can do, yeah